src/app/component/header/header.component.ts

```diff
@@ -21,13 +21,21 @@ export class HeaderComponent {
   }
 
   getData() {
```

Comment:
get data for what please specify like getUserData()

Reply:
i changed it to getUserdata()